lib/output.js

```diff
@@ -694,6 +694,7 @@ function OutputStream(options) {
         output.semicolon();
     });
     DEFPRINT(AST_Toplevel, function(self, output){
+        use_asm = false;
```

Comment:
Could you revert this change and use the fix outlined in https://github.com/mishoo/UglifyJS2/pull/1150#issuecomment-227139356 instead?